src/state/index.tsx

```diff
@@ -34,7 +36,7 @@ export default function AppStateProvider(props: React.PropsWithChildren<{}>) {
   const reporterToken = window.location.hash.substr(1);
 
   var endpoint = '';
-  fetch(`${process.env.PUBLIC_URL}/config.json`)
+  fetch(`${process.env.PUBLIC_URL}/config_mt_intg_01_aws_us_east_1.json`)
```

Comment:
This still needs to be config.json even though the file doesn't exist in the repository.
Please update the readme file (or create it if it doesn't exist) to inform developers that they need to create the public/config.json file from one of the examples there depending on the environment they wish to test against (and to possibly update the value of the url if it has been changed by the CICD)